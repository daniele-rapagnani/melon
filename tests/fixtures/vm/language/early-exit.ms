let f = |x| => {
    if (x > 0) {
        return x * 2;
    }

    if (x > 100) {
        return x - 100;
    }
};

io.print(f(10));
io.print(f(100));
io.print(f(-15));