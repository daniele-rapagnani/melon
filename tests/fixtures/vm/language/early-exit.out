20
200
null
